docs/guides/plugins.md

```diff
@@ -107,9 +107,10 @@ MyPlugin = crescent.Plugin[MyBot, None]
 
 typing.reveal_type(MyPlugin().app)  # `MyBot`
 ```
-## Model
+## Sharing Objects Between Files
```

Comment:
"Objects" feels like the wrong term. Maybe "State" instead?

"Files" -> "Plugins"? If we have no plugins, we can just reference the `client`, so the `model` is unneeded.

Reply:
Yeah, state is way better. I will use that.